web/src/connectors/task-annotator-connector/use-split-validation.ts

```diff
@@ -78,24 +81,54 @@ export default function useSplitValidation({
         { enabled: isSplitValidation }
     );
 
-    const userPages: AnnotationsByUserObj[] = useMemo(() => {
-        if (!byUser) {
+    const { data: pages } = useGetPageSummary(
+        { taskId: task?.id, taskType: task?.is_validation },
+        { enabled: Boolean(task) }
+    );
+
+    const { data: latestRevision } = useLatestAnnotationsByUser(
+        {
+            fileId,
+            jobId: job?.id,
+            pageNumbers: pages?.annotated_pages // передать сюда annotated из page sammury
```

Comment:
I believe comments should be in English

Reply:
I thought I forgot to push last commit from local, thanks